almond/DOCS.md

```diff
@@ -27,14 +27,11 @@ Got questions?
 
 You have several options to get them answered:
 
-- The [Home Assistant Discord Chat Server][discord].
-- The Home Assistant [Community Forum][forum].
-- Join the [Reddit subreddit][reddit] in [/r/homeassistant][reddit]
+- The [Almond Discord Chat Server][discord].
+- The Almond [Community Forum][forum].
 
 In case you've found a bug, please [open an issue on our GitHub][issue].
 
-[discord]: https://discord.gg/c5DvZ4e
-[forum]: https://community.home-assistant.io
-[issue]: https://github.com/home-assistant/hassio-addons/issues
-[reddit]: https://reddit.com/r/homeassistant
-[repository]: https://github.com/hassio-addons/repository
+[discord]: https://discord.gg/anthtR4
+[forum]: https://community.almond.stanford.edu
+[issue]: https://github.com/stanford-oval/almond-server/issues
```

Comment:
The core add-ons are part of Home Assistant, and thus should be handled here.

Please revert this change. It is OK to link to upstream, but we want to handle and collect issues here. The project carries the responsibility for these add-ons and will redirect users upstream when needed. Additionally, it is very confusing to guide people to different places for different add-ons.

Reply:
> We want users to reach out to upstream directly, to report issues and suggestions.

We don't. This was not a request, but a requirement of core integrations. Please adjust as requested.